tests/engine/adhoc_test.go

```diff
@@ -32,6 +32,7 @@ func runTests(t *testing.T, newEngine func() wasm.Engine) {
 	unreachable(t, newEngine)
 	memory(t, newEngine)
 	recursiveEntry(t, newEngine)
+	importedAndExportedFunc(t, newEngine)
```

Comment:
Can we rewrite here like the following? Do we intentionally avoid it or is it ok to do that?
```go

	t.Run("fibonacci", func(t *testing.T) {
		fibonacci(t, newEngine)
	})
	t.Run("fac", func(t *testing.T) {
		fac(t, newEngine)
	})
	t.Run("unreachable", func(t *testing.T) {
		unreachable(t, newEngine)
	})
	t.Run("memory", func(t *testing.T) {
		memory(t, newEngine)
	})
	t.Run("recursive entry", func(t *testing.T) {
		recursiveEntry(t, newEngine)
	})
	t.Run("imported-and-exported func", func(t *testing.T) {
		importedAndExportedFunc(t, newEngine)
	})
```

Reply:
sounds good